website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -20,6 +20,32 @@ upgrade flow.
 
 The `connect.enable_serverless_plugin` configuration option was removed. Lambda integration is now enabled by default.
 
+#### Deprecating authentication via token query parameter
+
+Providing a Consul ACL token in API requests using the `token` query parameter is deprecated and will be removed in Consul 1.17.
+Instead, the token should be provided using the `X-Consul-Token` header or with the Bearer scheme in the authorization header as described in the [API authentication documentation](/consul/api-docs/api-structure#authentication).
+	
+To check whether the `token` query parameter is in use, search your Consul agent logs for the message:
+
+```shell-session
+$ This request is using a token query parameter which is deprecated and will be removed in Consul 1.17
+```
```

Comment:
````suggestion
Instead, you should provide the token through the `X-Consul-Token` header or with the Bearer scheme in the authorization header as described in the [API authentication documentation](/consul/api-docs/api-structure#authentication).
	
Check whether you are using the `token` query parameter by searching your Consul agent logs for the message:

```shell-session hideClipboard
$ This request is using a token query parameter which is deprecated and will be removed in Consul 1.17
````
```

Reply:
"the token query param" rather than "a token query param"